### PR TITLE
Update glossary

### PR DESCRIPTION
**Pull request by tvararu**
Add CPD and TVS. CPD definition courtesy of @benilovj.

### CODE REVIEW

File: app/glossary.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ These are sums of money available to trainees on (fee-based) courses in certain
 
 ## C
 
+### Continuous Professional Development (CPD)
+
+Area around offering training for teachers.
```
Comment (frankieroberto) [inline app/glossary.md]: I don't think this term is teacher-specific? And should it be de-capitalised? 🤔 

```suggestion
### Continuous professional development (CPD)

A generic term referring to training of people already qualified and in employment. Avoid using the acronym, or spell it out on first use.
```

File: app/glossary.md [hunk 2]
```diff
@@ -247,6 +251,10 @@ Used to predict and manage the ITT market.
 
 Outstanding schools that work with others to provide high quality training to new and experienced staff. Cannot be lead schools or providers.
 
+### Teaching Vacancies (TVS)
+
+Service to [find and apply for teaching jobs](https://teaching-vacancies.service.gov.uk/).
```
Comment (frankieroberto) [inline app/glossary.md]:
```suggestion
A service for [advertising, finding and applying for teaching jobs](https://teaching-vacancies.service.gov.uk/). Capitalised. Do not use the acronym publicly.
```

**Review by frankieroberto — CHANGES_REQUESTED** [review]
Thanks for adding these! We should also add [National professional qualifications (NPQs)](https://www.gov.uk/government/publications/national-professional-qualifications-npqs-reforms/national-professional-qualifications-npqs-reforms) but that could be a separate PR.

The linter doesn't seem to run on forks, but hopefully that'll be fixed in #529.

File: app/glossary.md (hunk 2 above)
Comment (edwardhorsford) [reply app/glossary.md]: Should `vacancies` be capitalised?

Comment (frankieroberto) [reply app/glossary.md]: Possibly not, but it is on [the service](https://teaching-vacancies.service.gov.uk/).

**Comment by tvararu** [comment]
Cheers @edwardhorsford and @frankieroberto for the feedback! I didn't have notifications turned on which is why it took me days to commit the suggestions :sweat_smile:

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**